bootstrap/eksctl/crossplane/aws-provider.yaml

```diff
@@ -6,7 +6,7 @@ kind: Provider
 metadata:
   name: provider-aws
 spec:
-  package: "crossplane/provider-aws:v0.30.1"
+  package: "crossplane/provider-aws:v0.32.0"
```

Comment:
Should this be `xpkg.upbound.io/crossplane-contrib/provider-aws:v0.33.0`?

Reply:
Oh yeah you are correct. I was just looking at their DockerHub.